core/arch/arm/kernel/spmc_sp_handler.c

```diff
@@ -4,13 +4,20 @@
  */
 #include <assert.h>
 #include <bench.h>
+#include <io.h>
 #include <kernel/panic.h>
 #include <kernel/secure_partition.h>
 #include <kernel/spinlock.h>
 #include <kernel/spmc_sp_handler.h>
 #include <optee_ffa.h>
+#include <string.h>
 #include "thread_private.h"
 
+#define FFA_NW_ID			0
```

Comment:
Please use thread_get_tsd()->rpc_target_info instead of this define which hard codes the ID of normal world.